### DISCUSSION TITLE
userChromeJSManager: how to reset it in relation to the 'Tool Menu'

### DISCUSSION THREAD

**Discussion by logusergithub** [xiaoxiaoflood/firefox-scripts#172] [General Support] [answered]
Hi,
when I click on 'Settings > Switch to item in Tool Menu' the related option to return to the icon on the toolbar is not present in the 'Tool Menu'. Also, by uninstalling (Ctrl+left) or simply removing the script from the chrome folder the instructions for placement in the 'Tool Menu' remain unchanged: if I use it again in a new profile it goes directly to the 'Tool  Menu' (in addition to having the icon on the Toolbar).
Where that the script stores these settings that are global for all Firefox profiles? What am I missing?
Thanks.

**Comment by xiaoxiaoflood** [comment, marked as answer]
The option to revert exists:

![image](https://user-images.githubusercontent.com/25161346/188543333-3ac7758f-86ab-41f4-bb39-0df36abefee8.png)

I guess you're confusing Tools menu, which is part of Menu Bar (you usually can activate it by pressing just `Alt`), with the main menu (*hamburger* button in top-right).

userChromeJS menu in main menu is always present and doesn't handle button placement.

> Where that the script stores these settings that are global for all Firefox profiles?

Open `about:config` and look for `userChromeJS.showtoolbutton`.

**Comment by logusergithub** [comment]
@xiaoxiaoflood, thank you very much for the reply, you have guessed right... My goodness, I fell into a big confusion between the Tools Menu and the Main Menu and it didn't even cross my mind the Menu Bar by not using it for a long time. 
I was aware of the entries in about:config because I precisely tried everything after this misinterpretation, basically it started when I switched the option and then opened the Main Menu (if I had opened it earlier I would have realized it was already there).

Well, now that it's cleared up I seem to have found 2 ways to hide from the Main Menu:

- delete lines 433-464
- add "#appMenu-userChromeJS-button," to "Main-Hamburger.css" from SimpleMenuWizard

PS: thanks also for the "Extension Options Menu" of great use!
